etc/a.gwf

```diff
@@ -177,6 +192,10 @@ no_image_for_visitor=no
 # Notes = individual note or Marriage note.
 no_note_for_visitor=yes/no
 
+# A list of aliases for files containing notes (?? TODO check behaviour ??)
+# default notes.alias
```

Comment:
What do you want to check?

Also, what is the use case for this?

Seems like symlinks would be just fine. imho geneweb does not need this feature.